### PR TITLE
Add CVE-2024-40766

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**

### CODE REVIEW

**Review by ybadaoui-ostorlab — APPROVED** [review]
LGTM

File: tests/test-requirement.txt [hunk 1]
```diff
@@ -7,4 +7,5 @@ requests_mock
 types-requests
 pytest_mock
 ruff
-semantic_version
\ No newline at end of file
+semantic_version
```
Comment (3asm) [inline tests/test-requirement.txt:10]: Why is semantic_version in test requirements?

File: tests/test-requirement.txt [hunk 2]
```diff
@@ -7,4 +7,5 @@ requests_mock
...
+pysnmp==4.4.6
```
[the remark this answers is not in the document]
Comment (nmasdoufi-ol) [reply tests/test-requirement.txt:11]: Ok.

File: tests/test-requirement.txt (hunk 1 above)
Comment (nmasdoufi-ol) [reply tests/test-requirement.txt:10]: Ok.

**Review by 3asm — APPROVED** [review]

**Pull request merged**